### ISSUE TITLE
Storing products on client side vs querying

### ISSUE THREAD

**Issue by d21d3q** [ittus/vuejs-firebase-shopping-cart#5]
Is it optimal to store all products in (vuex) store? 
In case of haveing huge shop, storing thunders/thousands of products is not so optimal in sense of clients memory comsumption. How about lazy loading of products, and querying products from server based on users criteria?

**Comment by ittus** [comment]
You're right. We can do simple pagination and lazy load when querying products. Feel free to create a pull request 👍

**Comment by d21d3q** [comment]
There is also [topic on vue forum](https://forum.vuejs.org/t/vuex-to-store-all-my-products-and-to-or-not-to-split-out-api-requests/18034/2) related to this issue.
